### PR TITLE
Port docs for new Ping.SendAsync() methods

### PR DESCRIPTION
**Pull request by antonfirsov**
## Summary

Contributes to https://github.com/dotnet/runtime/issues/88569.

### CODE REVIEW

**Review by MihaZupan — APPROVED** [review]

File: xml/System.Net.NetworkInformation/Ping.xml [hunk 1]
```diff
@@ -2490,13 +2490,29 @@ The IP address returned by any of the <xref:System.Net.NetworkInformation.Ping.S
...
+        <param name="cancellationToken">
+        The token to monitor for cancellation requests. The default value is
+        <see cref="P:System.Threading.CancellationToken.None"/>
+        .
+        </param>
+        <summary>Sends an Internet Control Message Protocol (ICMP) echo message with the specified data buffer to the computer that has the specified
+<see cref="T:System.Net.IPAddress"/>
+, and receives a corresponding ICMP echo reply message from that computer as an asynchronous operation. This overload allows you to specify a time-out value for the operation, a buffer to use for send and receive, control fragmentation and Time-to-Live values, and a
+<see cref="T:System.Threading.CancellationToken"/>
+for the ICMP echo message packet.</summary>
```
Comment (carlossanlop) [inline xml/System.Net.NetworkInformation/Ping.xml:2514]:
```suggestion
        <param name="buffer">A <see cref="T:System.Byte"/> array that contains data to be sent with the ICMP echo message and returned in the ICMP echo reply message. The array cannot contain more than 65,500 bytes.</param>
        <param name="options">A <see cref="T:System.Net.NetworkInformation.PingOptions"/> object used to control fragmentation and Time-to-Live values for the ICMP echo message packet.</param>
        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is
        <see cref="P:System.Threading.CancellationToken.None"/>.</param>
        <summary>Sends an Internet Control Message Protocol (ICMP) echo message with the specified data buffer to the computer that has the specified <see cref="T:System.Net.IPAddress"/>, and receives a corresponding ICMP echo reply message from that computer as an asynchronous operation. This overload allows you to specify a time-out value for the operation, a buffer to use for send and receive, control fragmentation and Time-to-Live values, and a <see cref="T:System.Threading.CancellationToken"/> for the ICMP echo message packet.</summary>
```

File: xml/System.Net.NetworkInformation/Ping.xml [hunk 2]
```diff
@@ -2539,13 +2555,27 @@ The IP address returned by any of the <xref:System.Net.NetworkInformation.Ping.S
...
+        object used to control fragmentation and Time-to-Live values for the ICMP echo message packet.
+        </param>
+        <param name="cancellationToken">
+        The token to monitor for cancellation requests. The default value is
+        <see cref="P:System.Threading.CancellationToken.None"/>
+        .
+        </param>
+        <summary>Sends an Internet Control Message Protocol (ICMP) echo message with the specified data buffer to the specified computer, and receives a corresponding ICMP echo reply message from that computer as an asynchronous operation. This overload allows you to specify a time-out value for the operation, a buffer to use for send and receive, control fragmentation and Time-to-Live values, and a
+<see cref="T:System.Threading.CancellationToken"/>
+for the ICMP echo message packet.</summary>
```
Comment (carlossanlop) [inline xml/System.Net.NetworkInformation/Ping.xml:2577]:
```suggestion
        <param name="hostNameOrAddress"> The computer that is the destination for the ICMP echo message. The value specified for this parameter can be a host name or a string representation of an IP address. </param>
        <param name="timeout">The amount of time (after sending the echo message) to wait for the ICMP echo reply message.</param>
        <param name="buffer">A <see cref="T:System.Byte"/> array that contains data to be sent with the ICMP echo message and returned in the ICMP echo reply message. The array cannot contain more than 65,500 bytes.</param>
        <param name="options">A <see cref="T:System.Net.NetworkInformation.PingOptions"/> object used to control fragmentation and Time-to-Live values for the ICMP echo message packet.</param>
        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="P:System.Threading.CancellationToken.None"/>.</param>
        <summary>Sends an Internet Control Message Protocol (ICMP) echo message with the specified data buffer to the specified computer, and receives a corresponding ICMP echo reply message from that computer as an asynchronous operation. This overload allows you to specify a time-out value for the operation, a buffer to use for send and receive, control fragmentation and Time-to-Live values, and a <see cref="T:System.Threading.CancellationToken"/> for the ICMP echo message packet.</summary>
```

**Review by carlossanlop — APPROVED** [review]
Some spacing suggestions but otherwise looks good.

**Pull request merged**